### DISCUSSION TITLE
Strengh control to deforum

### DISCUSSION THREAD

**Discussion by slimanibenbadis** [Rakile/deforumation#11] [General]
Hello, 
First, I wanted to say that your work is really cool and allows for more amazing creations and animations, so thank you !

My main question is, do we have the possibility to let the strengh and movement control to deforum and use deforumation mainly as an editing tool to tweak prompts and reverse back to previous images if the render of the prompt does not fit ? 

For exemple, using it in combination with parsec, plan the video rythm and dynamic through parsec and then using deforumation to check live the renderings and tweak thing in real time.

Thanks for your answers 

Keep up the good work :) !

**Comment by lainollll** [comment]
"My main question is, do we have the possibility to let the strengh and movement control to deforum and use deforumation mainly as an editing tool to tweak prompts and reverse back to previous images if the render of the prompt does not fit ?". This is basically how it works., you have the checkbox near the strength slider." use deforumation strength". Uncheck that and it uses the strenghts from deforum.  And all movements that is keyframes in deforum should just slip by deforumation. Deforumation adds movement to the scheduled keyframes in deforum.    And at the top you have checkbox if you want the deforum scheduled prompts be used.

**Comment by lainollll** [comment]
What is parsec?

**Reply by slimanibenbadis** [reply]
Sorry I made a mistake writing it, it is "parseq" or sd-parseq https://sd-parseq.web.app/ 
It is a tool to plan a deforum sequence and edit the different values and parameters
It produces a file at the end that can be pasted into deforum in the parseq section dedicated, and use it as guidance for the sequence.
I must add that I am not the author of this tool, and I am not affiliated with it, just using it sometimes :)

**Reply by lainollll** [reply]
I se no problem using it as you explain, everything you can schedule in deforum should pass through deforumation.  Where you can tweak.   But you won't have the ability to change deforum scheduled prompts.  Deforumation is not about automating animations,  it's for creating animations.  But you can use it for those purposes you say.
